tests/acceptance/features/apiProvisioningLDAP/groups.feature

```diff
@@ -0,0 +1,59 @@
+@api
+Feature: manage groups
+  As an administrator
+  I want to be able to add, delete and modify groups via the Provisioning API
+  So that I can easily manage groups when user LDAP is enabled
+
+  Background:
+    Given using OCS API version "2"
+    Given user "brand-new-user" has been created with default attributes
+
+  @smokeTest
+  Scenario Outline: admin creates a group
+    When the administrator sends a group creation request for group "<group_id>" using the provisioning API
+    Then the OCS status code should be "200"
+    And the HTTP status code should be "200"
+    And group "<group_id>" should exist
+    Examples:
+      | group_id    | comment                     |
+      | simplegroup | nothing special here        |
+      | España      | special European characters |
+      | नेपाली      | Unicode group name          |
+
+  @smokeTest
+  Scenario Outline: adding a user to a group
+    Given group "<group_id>" has been created
+    When the administrator adds user "brand-new-user" to group "<group_id>" using the provisioning API
+    Then the OCS status code should be "200"
+    And the HTTP status code should be "200"
```

Comment:
It's better to add `And user "brand-new-user" should belong to group "<group_id>"`